third_party/libusb/webport/src/libusb-to-webusb-adaptor-unittest.js

```diff
@@ -315,5 +375,95 @@ goog.exportSymbol('testLibusbToWebusbAdaptor', {
         configurations);
   },
 
+  // Test the `getConfigurations()` method against the data obtained from a real
+  // SCM SCR 3310 device.
+  'testGetConfigurations_scmScr3310': async function() {
+    const EXTRA_DATA_TRANSFER_PARAMS = {
+      'requestType': 'standard',
+      'recipient': 'device',
+      'request': 0x06,
+      'value': 0x200,
+      'index': 0
+    };
+
+    // Arrange:
+    const device = deepClone(SCM_SCR_3310_DEVICE);
+    device['open'] = goog.testing.createFunctionMock();
+    device['open']().$do(async () => {});
```

Comment:
Should this be does instead of do (also in line 395)? If not, should it be do instead of does in line 400/408? If both are accurate, could you explain the difference to me, please?

Reply:
Good catch, it should be "does" indeed. Apparently, the framework has both, and they have some subtle difference. I suspect the wrong one was working accidentally, because we pass an empty function.